local/gugcat/classes/grade_aggregation.php

```diff
@@ -167,6 +167,7 @@ public static function get_rows($course, $modules, $students, $showerrors = fals
                     $grdobj->is_imported = !is_null($pg) ? true : false;
                     $grdobj->is_child = local_gugcat::is_child_activity($item) ? true : false;
                     $grdobj->grade = $grade;
+                    $grdobj->orignalgrade = is_null($scaleid) ? $grade : $grdvalue;
```

Comment:
Please check the spelling - orignal

Reply:
Fixed in commit https://github.com/uofg-andf/andfmoodle/pull/479/commits/c540f4133a7d9a66e7fa71f459ca55d298c60264